src/overmap.cpp

```diff
@@ -1761,6 +1769,81 @@ bool overmap::generate_sub( const int z )
     return requires_sub;
 }
 
+bool overmap::generate_over( const int z )
+{
+    bool requires_over = false;
+    std::vector<point> bridge_points;
+
+    // These are so common that it's worth checking first as int.
+    const oter_id skip_below[5] = {
+        oter_id( "empty_rock" ), oter_id( "forest" ), oter_id( "field" ),
+        oter_id( "forest_thick" ), oter_id( "forest_water" )
+    };
+
+    for( int i = 0; i < OMAPX; i++ ) {
+        for( int j = 0; j < OMAPY; j++ ) {
+            tripoint p( i, j, z );
+            const oter_id oter_below = ter( p + tripoint_below );
+            const oter_id oter_ground = ter( tripoint( p.xy(), 0 ) );
+
+            // implicitly skip skip_below oter_ids
+            bool skipme = false;
+            for( const oter_id &elem : skip_below ) {
```

Comment:
If you change `skip_below ` into a `std::array`, you can check its content via
```C++
if( std::find( skip_below.begin(), skip_below.end(), oter_below ) != skip_below.end() ) { // it's contained.
```
Or make it a `std::set` and checking that is even easier.

Reply:
Nice!  Fixed, thanks.